.github/actions/test-install-components/install_component.sh

```diff
@@ -22,8 +22,16 @@ else
     exit 1
 fi
 
-if [ "${ARCH}" = "i386" ] || [ "${ARCH}" = "armv7hl" ]; then
+if [ "${ARCH}" = "i386" ] || [ "${ARCH}" = "armhf" ]; then
     linux="linux32"
+    if [ "${ARCH}" = "armhf" ] && [ "${SYSTEM}" = "rpm" ]; then
+        install="rpm -ivh --force --ignorearch"
+        WAZUH_MANAGER="10.0.0.2" $linux $install "/packages/$package_name"| tee /packages/status.log
+        if [ "$(rpm -qa | grep wazuh-agent)" ]; then
+            echo " installed wazuh-agent" >> /packages/status.log
+            exit 0
+        fi
```

Comment:
If the installation is unsuccessful, we should exit with an error to avoid executing the rest of the code.